### PR TITLE
Question answer on modal

### PR DESCRIPTION
**Pull request by nori3506**

### CODE REVIEW

File: client/src/screens/Product.js [hunk 1]
```diff
@@ -72,6 +72,7 @@ export default () => {
     try {
       await addAnswer(token, params)
       setAnswer({})
+      setQuestion('')
```
Comment (satoshinbp) [inline client/src/screens/Product.js:75]: question is not string type

File: client/src/screens/Product.js [hunk 2]
```diff
@@ -320,11 +331,13 @@ export default () => {
...
 
-      <Button variant="fab">Ask a Question</Button>
+      <Button variant="fab" onPress={() => setQuestionForm()}>
+        Ask a Question
+      </Button>
 
       <SlideModal
         isOpen={isModalOpen}
-        onClose={() => setIsModalOpen(false)}
+        onClose={() => closeModal()}
```
Comment (satoshinbp) [inline client/src/screens/Product.js:340]:
```suggestion
        onClose={closeModal}
```

File: client/src/screens/Product.js [hunk 3]
```diff
@@ -208,13 +209,23 @@ export default () => {
...
   const setReportForm = (item, fixedquestionIndex, answerIndex) => {
     setIsModalOpen(true)
     setModalContentType('report')
 
     setReportItem({ fixedQandAsId: item._id, fixedquestionIndex, answerIndex })
   }
 
+  const closeModal = () => {
+    setIsModalOpen(false)
+    setQuestion('')
```
Comment (satoshinbp) [inline client/src/screens/Product.js:226]: question is not string type

**Review by satoshinbp — CHANGES_REQUESTED** [review]

File: client/src/screens/Product.js (hunk 3 above)
Comment (nori3506) [reply client/src/screens/Product.js:226]: When we setQuestion({}) here, then error occurs!

Comment (satoshinbp) [reply client/src/screens/Product.js:226]: how about null?

Comment (nori3506) [reply client/src/screens/Product.js:226]: let me

Comment (nori3506) [reply client/src/screens/Product.js:226]: umm buggy
only "" works..

**Pull request merged**